app/models/tax_form_due_dates/dropdown.rb

```diff
@@ -0,0 +1,68 @@
+module TaxFormDueDates
+  module Dropdown
+     extend ActiveSupport::Concern
```

Comment:
this should not be saved in a concern. try a lib module instead please. read up on what model concerns do in the docs